dashboard/src/components/AppView/AppView.test.tsx

```diff
@@ -241,22 +226,26 @@ describe("AppViewComponent", () => {
     it("renders an URL table if an Ingress exists", () => {
       const wrapper = shallow(<AppViewComponent {...validProps} />);
       const ingress = {
-        metadata: {
-          name: "foo",
-        },
-        spec: {
-          rules: [
-            {
-              host: "foo.bar",
-              http: {
-                paths: [{ path: "/ready" }],
+        isFetching: false,
+        item: {
+          metadata: {
+            name: "foo",
+          },
+          spec: {
+            rules: [
+              {
+                host: "foo.bar",
+                http: {
+                  paths: [{ path: "/ready" }],
+                },
               },
-            },
-          ],
-        } as IIngressSpec,
-      } as IResource;
+            ],
+          } as IIngressSpec,
+        } as IResource,
```

Comment:
this is quite messy, ideally we should just be able to use IIngress here and not have to convert it twice 
(once for the spec, and for the full thing)